python/pdo/contract/response.py

```diff
@@ -90,6 +100,216 @@ class ContractResponse(object) :
     Class for managing the contract operation response from an enclave service
     """
 
+    __start_threads_for_commit__ = True
+    transaction_submission_exceptions_queue = queue.Queue()
+
+    transaction_executor = concurrent.futures.ThreadPoolExecutor(max_workers=1) # executor that submit transactions
+    pending_transactions_queue = queue.Queue()
+
+    txn_completion_condition = threading.Condition()
+    set_of_completed_txns = set()
+    
```

Comment:
Does this really belong here? It seems like moving all of the transaction submission handling into a separate module that is always started through some initialization process would be a lot simpler. And would allow us to keep classes more single purpose. This adds a lot of complexity to something which was never intended to be complex.

Reply:
Ok, I would like to discuss this. Ideally, I would prefer a separate class for all of the transacction processing, which means moving a lot of the existing code, as well the new code added. Would you want me to discuss this with you and others, and make a separate PR for this? (I prefer a separate PR)